salt/cloud/clouds/vsphere.py

```diff
@@ -867,3 +867,60 @@ def snapshot_list(kwargs=None, call=None):
             ret[vm]['snapshots'].append(snap.get_name())
 
     return ret
+
+
+def create_snapshot(kwargs=None, call=None):
+    '''
+    Create a snapshot
+
+    @name: Name of the virtual machine to snapshot
+    @snapshot: Name of the snapshot
+    @description: Description of the snapshot (optional)
+    @memory: Dump of the internal state of the virtual machine (optional)
+
```

Comment:
For releases that haven't been forked from develop yet, you can tag it with the codename, which in this case is beryllium, so `.. versionadded:: Beryllium` should be sufficient, thanks.